### PR TITLE
hexdump: add Russian translation

### PR DESCRIPTION
**Pull request by trolzen**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages.ru/common/hexdump.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# hexdump
+
+> Дамп файла в ASCII, десятичном, шестнадцатеричном и восьмеричном форматах.
+> Больше информации: <https://manned.org/hexdump>.
+
+- Распечатать шестнадцатеричное представление файла, заменяя повторяющиеся строки на '*':
+
+`hexdump {{файл}}`
```
Comment (EmilySeville7cfg) [inline pages.ru/common/hexdump.md:8]: Syntax is incorrect both in the original page and here. Is should be (it's assumed that path to file is expected and not just a plain filename):

```suggestion
`hexdump {{путь/к/файлу}}`
```

File: pages.ru/common/hexdump.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Отобразить шестнадцатеричное и ASCII представление в две колонки:
+
+`hexdump -C {{файл}}`
```
Comment (EmilySeville7cfg) [inline pages.ru/common/hexdump.md:12]: Syntax is incorrect both in the original page and here. Is should be (it's assumed that path to file is expected and not just a plain filename):

```suggestion
`hexdump -C {{путь/к/файлу}}`
```

File: pages.ru/common/hexdump.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- Отобразить двухколончатое представление файла, обработав только указанное число байтов с начала:
+
+`hexdump -C -n{{количество_байтов}} {{файл}}`
```
Comment (EmilySeville7cfg) [inline pages.ru/common/hexdump.md:16]: Syntax is incorrect both in the original page and here. Is should be (it's assumed that path to file is expected and not just a plain filename):

```suggestion
`hexdump -C -n{{количество_байтов}} {{путь/к/файлу}}`
```

File: pages.ru/common/hexdump.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+- Не заменять повторяющиеся строки на '*':
+
+`hexdump --no-squeezing {{файл}}`
```
Comment (EmilySeville7cfg) [inline pages.ru/common/hexdump.md:20]: Syntax is incorrect both in the original page and here. Is should be (it's assumed that path to file is expected and not just a plain filename):

```suggestion
`hexdump --no-squeezing {{путь/к/файлу}}`
```

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

**Comment by trolzen** [comment]
@EmilySeville7cfg I've updated this PR with your suggestions, but I used the recommended form from the `common-arguments.md`

File: pages.ru/common/hexdump.md [hunk 5]
```diff
@@ -5,16 +5,16 @@
 
 - Распечатать шестнадцатеричное представление файла, заменяя повторяющиеся строки на '*':
 
-`hexdump {{файл}}`
+`hexdump {{путь/до/файла}}`
```
Comment (EmilySeville7cfg) [inline pages.ru/common/hexdump.md:8]: The [original](https://github.com/tldr-pages/tldr/blob/main/pages/common/hexdump.md) page should be updated too to reflect this change.

File: pages.ru/common/hexdump.md [hunk 6]
```diff
@@ -5,16 +5,16 @@
...
 
 - Отобразить шестнадцатеричное и ASCII представление в две колонки:
 
-`hexdump -C {{файл}}`
+`hexdump -C {{путь/до/файла}}`
```
Comment (EmilySeville7cfg) [inline pages.ru/common/hexdump.md:12]: The [original](https://github.com/tldr-pages/tldr/blob/main/pages/common/hexdump.md) page should be updated too to reflect this change.

File: pages.ru/common/hexdump.md [hunk 7]
```diff
@@ -5,16 +5,16 @@
...
 
 - Отобразить двухколончатое представление файла, обработав только указанное число байтов с начала:
 
-`hexdump -C -n{{количество_байтов}} {{файл}}`
+`hexdump -C -n{{количество_байтов}} {{путь/до/файла}}`
```
Comment (EmilySeville7cfg) [inline pages.ru/common/hexdump.md:16]: The [original](https://github.com/tldr-pages/tldr/blob/main/pages/common/hexdump.md) page should be updated too to reflect this change.

File: pages.ru/common/hexdump.md [hunk 8]
```diff
@@ -5,16 +5,16 @@
...
 
 - Не заменять повторяющиеся строки на '*':
 
-`hexdump --no-squeezing {{файл}}`
+`hexdump --no-squeezing {{путь/до/файла}}`
```
Comment (EmilySeville7cfg) [inline pages.ru/common/hexdump.md:20]: The [original](https://github.com/tldr-pages/tldr/blob/main/pages/common/hexdump.md) page should be updated too to reflect this change.

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
- No issues found but merging blocked to update the original page at the first place. PR will be approved after the original page is refreshed (I've already [sent](https://github.com/tldr-pages/tldr/pull/9520) a PR for it).
- Thanks for a contribution. 🚀

**Review by EmilySeville7cfg — APPROVED** [review]
- Approved because all issues are already addressed to.
- This page should be reviewed by `hexdump` users.
- Thanks for a contribution. 😸

**Pull request merged**